hack/docgen/README.md

```diff
@@ -1,12 +1,16 @@
 # Cluster Monitoring Operator Docgen
 
-This package contains the code that automatically generates `Documentation/api.md` the code is heavily inspired by the (current) prometheus-operator codegen solution however, some adaptations were made to the format to alight it as much as possible with the OpenShift docs format.
+This package contains the code that automatically generates `Documentation/api.md` and `Documentation/resources.md`. The code is heavily inspired by the prometheus-operator's codegen stool however, some adaptations were made to the format to align it as much as possible with the OpenShift docs format.
 
-### Goal
-The goal of this code is to automate the generation of documentation that will not only be used by users but also by our technical writers who have to maintain an updated version of it on OpenShift docs. Because of this requirement, a custom code solution was used instead of an open source one to facilitate compliance with the OpenShift docs format. 
+## Goal
 
-### How it works
-The code will generate the documentation based on comments in the golang code. It will ingest the file `pkg/manifests/types.go` which contains all the types used by CMO and parse it accordingly.
+The goal of this code is to automate the generation of documentation that will not only be used by users but also by our technical writers who have to maintain an updated version of it on OpenShift docs. Because of this requirement, a custom code solution was used instead of an open source one to facilitate compliance with the OpenShift docs format.
```

Comment:
```suggestion
The goal of this code is to automate the generation of documentation that will not only be used by users but also by our technical writers who have to maintain an updated version of it for the OpenShift docs. Because of this requirement, a custom code solution was used instead of an open source one to facilitate compliance with the OpenShift docs format.
```